content/docs/secure/encryption/vault.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "Vault Configuration"
+linkTitle: "Vault Configuration"
+weight: 3
+Description: >
+  Configuration requirements for Vault server.
+---
+
+## Vault Server Installation
+
+If there is already a Vault server available, skip to [Minimum Server Configuration](#minimum-server-configuration).
+
+If there is no Vault server available to use with Encryption, it can be installed in many ways following [Hashicorp Vault documentation](https://www.vaultproject.io/docs).
+
+For testing environment, however, a simple deployment suggested in this section may suffice. 
+It creates a standalone server with in-memory (non-persistent) storage, running in a Docker container.
+
+> **NOTE**: With in-memory storage, the encryption keys are permanently destroyed upon the server termination.
+
+#### Generate TLS certificates for server and client
+
+Create server CA private key and certificate:
+
+```shell
+openssl req -x509 -sha256 -days 365 -newkey rsa:2048 -nodes \
+	-subj "/CN=Vault Root CA" \
+	-keyout server-ca.key \
+	-out server-ca.crt
+```
+
+Create server private key and CSR:
+
+```shell
+openssl req -newkey rsa:2048 -nodes \
+	-subj "/CN=vault-demo-server" \
+	-keyout server.key \
+	-out server.csr
+```
+
+Create server certificate signed by the CA:
+
+> Replace `10.11.12.13` below with an IP address by which Encryption can reach the Vault server. 

```

Comment:
Please mask the IP address or use a variable name.

Reply:
Addressed.